scripts/commands.js

```diff
@@ -68,6 +68,7 @@ program
   .option('--disable_brave_extension', 'disable loading the Brave extension')
   .option('--disable_pdfjs_extension', 'disable loading the PDFJS extension')
   .option('--enable_brave_update', 'enable brave update')
+  .option('--channel <target_chanel>', 'target channel to start', /^(beta|dev|nightly|release)$/i, 'release')
```

Comment:
I wonder if the default here should be nightly? I won't block on that though just a question for a follow up.

Reply:
Yep, I think `nightly` can be default by applying this to `build` command.